sdks/python/container/boot.go

```diff
@@ -210,15 +224,29 @@ func main() {
 	wg.Add(len(workerIds))
 	for _, workerId := range workerIds {
 		go func(workerId string) {
+			defer wg.Done()
 			log.Printf("Executing: python %v", strings.Join(args, " "))
-			log.Fatalf("Python exited: %v", execx.ExecuteEnv(map[string]string{"WORKER_ID": workerId}, "python", args...))
-			wg.Done()
+			log.Printf("Python exited: %v", execx.ExecuteEnv(map[string]string{"WORKER_ID": workerId}, "python", args...))
```

Comment:
This change was necessary for clean shutdown of the worker. IMHO, the previous implementation was erroneous, since the first thread to finish would system.exit() the whole parent process, even if other sibling processes were still running.